apps/web/.env.example

```
@@ -5,6 +5,9 @@ NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=YOUR_SUPABASE_PUBLISHABLE_KEY
 SUPABASE_SECRET_KEY=YOUR_SUPABASE_SECRET_KEY
 
 # Optional API keys
+# AI_GATEWAY_API_KEY authenticates AI Gateway requests used for AI credit packs/features.
+# Required in environments where AI credit purchase/deduction flows are enabled (including production).
+AI_GATEWAY_API_KEY=YOUR_AI_GATEWAY_API_KEY
 OPENAI_API_KEY=YOUR_OPENAI_API_KEY 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)